rspec-core.gemspec

```diff
@@ -13,6 +13,14 @@ Gem::Specification.new do |s|
   s.summary     = "rspec-core-#{RSpec::Core::Version::STRING}"
   s.description = "BDD for Ruby. RSpec runner and example groups."
 
+  s.metadata = {
+    'bug_tracker_uri'   => 'https://github.com/rspec/rspec-core/issues',
+    'changelog_uri'     => 'https://github.com/rspec/rspec-core/blob/master/Changelog.md',
+    'documentation_uri' => 'https://relishapp.com/rspec/rspec-core/docs',
```

Comment:
This is should be `http://rspec.info/documentation/` or `http://rspec.info/documentation/3.8/rspec-core` which mirrors how the existing link points to rdoc.

Reply:
I thought about that, but that misses the opportunity to loop in the relish docs too.